webapp/src/js/services/index.js

```diff
@@ -3,6 +3,8 @@
 
   angular.module('inboxServices', ['ngResource']);
 
+  require('./exception-handler');
+  require('./feedback');
```

Comment:
This file should be sorted alphabetically to make it easy to find stuff and not conflict with other changes.